src/nautilus_librarian/mods/dvc/domain/diff/renamed_path.py

```diff
@@ -0,0 +1,23 @@
+from nautilus_librarian.mods.dvc.domain.diff.path import Path
+from nautilus_librarian.mods.filesystem.domain.relative_filepath import RelativeFilepath
+
+
+class RenamedPath(Path):
+    def __init__(self, new_path: str, old_path: str) -> None:
+        self.path = RelativeFilepath(new_path)
+        self.old_path = RelativeFilepath(old_path)
+
+    def __str__(self) -> str:
+        return str(self.path)
```

Comment:
It seems strange to me that the string conversion conceals the fact that is a renaming path removing the old path. In my opinion, this can produce bugs difficult to locate.